subworkflows/local/hic_bwamem2.nf

```diff
@@ -68,7 +67,7 @@ workflow HIC_BWAMEM2 {
         .map { file ->
             tuple (
                 [
-                id: file[0].toString().split('/')[-1].split('_')[0] + '_' + file[0].toString().split('/')[-1].split('_')[1]
```

Comment:
@yumisims can you have a look at this line please if it's ok to change?
It fixes the warning currently raised in dev for both hap1 and hap2:
<pre>
WARN: Input tuple does not match tuple declaration in process `SANGERTOL_GENOMEASSEMBLY:GENOMEASSEMBLY:HIC_MAPPING_HAP2:HIC_BWAMEM2:CRAM_FILTER_ALIGN_BWAMEM2_FIXMATE_SORT` -- offending value: [[id:baUndUnlc1], /lustre/scratch123/tol/resources/nextflow/test-data/Undibacterium_unclassified/genomic_data/baUndUnlc1/hic-arima2/41741_2#7.sub.cram, /lustre/scratch123/tol/resources/nextflow/test-data/Undibacterium_unclassified/genomic_data/baUndUnlc1/hic-arima2/41741_2#7.sub.cram.crai, 0, 381, 41741_2#7.sub, 0, @RG\tID:41741_2#7\tDT:2021-11-01T00:00:00+0000\tPU:211101_A00495_0659_AHLYY7DSX2_2#7\tLB:51690611\tPG:SCS\tSM:SAMEA8580186\tCN:SC\tPL:ILLUMINA\tDS:ERP129860: Sequencing and assembly of symbiont/host genomes in marine and freshwater species, /lustre/scratch124/tol/projects/darwin/users/kk16/development/test/20240801_hic/workdir/52/463d3e252766af4596368243c43817/bwamem2/baUndUnlc1.asm.hic.hap2.fa, /lustre/scratch124/tol/projects/darwin/users/kk16/development/test/20240801_hic/workdir/b8/24f92d314b609be0bbb87de514df29/baUndUnlc1.asm.hic.hap2.fa]
</pre>

Reply:
looks fine to me. I don't recall I have this warning when I was working on it.